src/bp/ui-studio/src/web/views/OneFlow/diagram/VariablesEditor/index.tsx

```diff
@@ -93,10 +95,12 @@ const VariablesEditor: FC<Props> = props => {
   const renderTableRows = () => {
     return (
       <Fragment>
-        {props.currentFlow?.variables?.map((v, i) => (
+        {props?.currentFlow?.variables?.map((v, i) => (
```

Comment:
Curious about the added ? , can this ever be undefined?

Reply:
Probably not I probably added it there with the other null checks